teradatalabs/mapr52-hive/files/wardenTracker.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+hname=$(hostname)
+
+Services=0
+
+maprcliReady=$(maprcli service list -node $hname | grep 'ERROR (10009)' | wc -l)
+
+# WAIT FOR CLDB TO START
+while [ $Services -ne 2 ]
+do
+if [ $maprcliReady == 1 ]
+then
+   maprcliReady=$(maprcli service list -node $hname | grep 'ERROR (10009)' | wc -l)
+   Services=0
+else
+   Services=$(maprcli service list -node $hname | grep CLDB |awk '{$1=$1};1' | tr ' ' '\n' | tail -1f)
```

Comment:
Actually, the 6 lines containing the dockerize calls waiting for the services should just go to `bootstrap.sh`, this file should be deleted